src/components/Example/Example.js

```diff
@@ -1,18 +1,36 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import { Icon } from 'carbon-components-react';
+// import { Icon } from 'carbon-components-react';
 
 export default class Example extends React.Component {
   static propTypes = {
     children: PropTypes.node,
     title: PropTypes.string,
+    description: PropTypes.string,
     text: PropTypes.string,
-    correct: PropTypes.bool,
+    correct: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
+  };
+
+  renderCaption = (title, description) => {
```

Comment:
Just an idea; The signature of this method is close to React's function-based components. Does this signal that we can extract this portion as a separate component - If it's used only in `<Example>`, probably a component that is not exported?